packages/node_modules/@webex/plugin-meetings/src/constants.js

```diff
@@ -90,11 +92,13 @@ export const RTC_CONFIGURATION_FIREFOX = {iceServers: [], bundlePolicy: 'max-com
 export const RTC_CONFIGURATION = {iceServers: []};
 export const RESOURCE_ROOM = 'RESOURCE_ROOM';
 export const READY = 'ready';
+export const RECVONLY = 'RECVONLY';
```

Comment:
Should we call this `RECEIVE_ONLY` to be consistent with our own conventions? Or let this implementation detail remain so it's more obvious?

Reply:
i can do RECEIVE ONLY thats fine